lib/components/ms-input/MsSearchInput.vue

```diff
@@ -17,6 +18,8 @@
       :placeholder="$msTranslate(placeholder)"
       :clear-input="true"
       @ion-input="onChange($event.target.value)"
+      @ion-focus="onFocus"
```

Comment:
Same remark as @Max-7 on `MsTextArea`
> Could just do @ion-focus="hasFocus = true" and @ion-blur="hasFocus = false" just to avoid creating the functions.